style/base.css

```diff
@@ -17,3 +17,10 @@
   outline-offset: -3px;
   outline-style: auto;
 }
+
+.nebari-DeployAppIcon {
```

Comment:
The CSS needs to be fixed - currently it only displays when I give fixed size (e.g. `16px`).